src/components/MoneyTransaction.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import styles from './MoneyTransaction.module.css'
+import { action } from '@storybook/addon-actions'
+import { Button } from './Button'
+import { Table } from './Table'
+import { CreateEntry } from './Form'
+import { Heading, HeadingDisabled } from './Text'
+
+export const MoneyTransaction = () => {
+  return (
+        <div className = {`${styles.page}`}>
+            <div className = {`${styles.right}`}><Button onClick={action('clicked')}>Click me</Button></div>
```

Comment:
There is something wrong with your linter setup as this does not look correct to me. I won't deduct any points as `npm run lint` does not return any errors. But please try to reinstall eslint.